xapian-applications/omega/index_file.cc

```diff
@@ -1093,7 +1096,16 @@ index_mimetype(const string & file, const string & urlterm, const string & url,
 	    keywords = atomparser.keywords;
 	    // FIXME: topic = atomparser.topic;
 	    author = atomparser.author;
-	} else {
+	}
+   else if ( mimetype=="application/zip"){
+
+
+
+
+
+   }
+
+   else {
```

Comment:
This is poorly formatted, but a special case for `application/zip` here shouldn't be needed anyway -